cfme/containers/image.py

```diff
@@ -22,6 +25,18 @@
 )
 
 
+def run_smart_state_analysis(self):
```

Comment:
Can this run against a single image?
It seems to me that this should be a classmethod on Image
Possibly even two methods, one that says ```run_smart_state_analysis```
And the other would be a class method that was called ```run_smart_state_analysis_all_images``` or something like that.
Regardless, I think this should be a method of the Image class, perhaps we need a mixin for SSA at some point